src/arrayMethodReduce.js

```diff
@@ -5,7 +5,16 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue === undefined ? undefined : initialValue;
```

Comment:
are you sure you need this condition?

Reply:
I can do it through the initialValue, but the linter doesn't miss me. So I decided so

Reply:
what linter says you, attach screenshot, please

Reply:
![image](https://user-images.githubusercontent.com/50458410/63837004-4f4f3300-c983-11e9-86a0-7f3742169e6a.png)
But I disabled this error and added a new solution to GitHub.

Reply:
back to your prev solution, this is what was meant to edit
`let accumulator = initialValue === undefined ? undefined : initialValue;`
--->
`let accumulator = initialValue;`